python/read_credentials.py

```diff
@@ -4,14 +4,17 @@
 import csv
 
 def read():
-    with open('../credentials.txt') as file:
+    with open('credentials.txt') as file:
```

Comment:
I'd say yaml is more commonly used than plaintext for storing this kind of thing.